docker-compose-generator/docker-fragments/bitcoin-lnd.yml

```diff
@@ -57,6 +57,15 @@ services:
     expose:
       - "3000"
 
+  bitcoin_thub:
```

Comment:
Can you do a separate file for this - `opt-add-thunderhub.yml` and then include it in readme.md under `Available `BTCPAYGEN_ADDITIONAL_FRAGMENTS` currently are:`



At the bottom of the yml file, add 
```yml
required:
  - "bitcoin-lnd"
```

Reply:
Done! 👍